Sources/MessagingPush/MessagingPushInstance.swift

```diff
@@ -10,6 +10,7 @@ import UserNotifications
 // 1. Cannot be used in an App Extension so therefore, we cannot guarantee that the function
 //    exists in the SDK code at compile time.
 public protocol MessagingPushInstance: AutoMockable {
+    var moduleConfig: MessagingPushConfigOptions { get }
```

Comment:
By adding this line here, does it mean that it's officially part of the public API? 

Assuming it's part of the public API, could we have it added to APITest?

Reply:
Not officially though. I don't think we'll be recommending customers ever to use config from here. But I have added to `APITest` so we know whenever this changes.